src/axom/quest/examples/shaping_driver.cpp

```diff
@@ -642,6 +650,8 @@ int main(int argc, char** argv)
                                           shape.getMaterial(),
                                           shapeFormat)));
 
+    const klee::Dimensions shapeDim = shape.getGeometry().getInputDimensions();
```

Comment:
A key contribution of this PR is that, as we loop through the shapes in a Klee input, we can get the per-shape dimension, rather than being limited to the global dimension of the shapeset (`ShapeSet::getDimensions()`)